media/js/libs/mozilla-traffic-cop.js

```diff
@@ -132,7 +132,7 @@ Mozilla.TrafficCop.prototype.init = function() {
  */
 Mozilla.TrafficCop.prototype.initiateCustomCallbackRoutine = function() {
     // set a cookie to remember the chosen variation
-    Mozilla.Cookies.setItem(this.id, this.chosenVariation || Mozilla.TrafficCop.noVariationCookieValue, this.cookieExpiresDate);
+    Mozilla.Cookies.setItem(this.id, this.chosenVariation || Mozilla.TrafficCop.noVariationCookieValue, this.cookieExpiresDate, undefined, undefined, false, "lax");
```

Comment:
Nit: single quotes here and below
```suggestion
    Mozilla.Cookies.setItem(this.id, this.chosenVariation || Mozilla.TrafficCop.noVariationCookieValue, this.cookieExpiresDate, undefined, undefined, false, 'lax');
```